### PR TITLE
update docs

### PR DESCRIPTION
**Pull request by hal909**

### CODE REVIEW

File: contracts/truefi2/strategies/CurveYearnStrategy.sol [hunk 1]
```diff
@@ -105,7 +107,10 @@ contract CurveYearnStrategy is UpgradeableClaimable, ITrueStrategy {
         curveGauge.deposit(yBalance);
     }
 
-    /// @dev pull at least `minAmount` of tokens from strategy
+    /** @dev pull at least `minAmount` of tokens from strategy
```
Comment (yuchenlintt) [inline contracts/truefi2/strategies/CurveYearnStrategy.sol:110]: nitpick: first line of comment should just be `/**`

File: contracts/truefi2/strategies/CurveYearnStrategy.sol [hunk 2]
```diff
@@ -122,8 +127,9 @@ contract CurveYearnStrategy is UpgradeableClaimable, ITrueStrategy {
         transferAllToPool();
     }
 
-    /// @dev withdraw everything from strategy
-    /// Use with cautious because Curve slippage is not contolled
+    /** @dev withdraw everything from strategy
```
Comment (yuchenlintt) [inline contracts/truefi2/strategies/CurveYearnStrategy.sol:130]: nitpick: first line should be `/**`

Comment (hal909) [reply contracts/truefi2/strategies/CurveYearnStrategy.sol:130]: resolved

File: test/truefi2/Liquidator2.test.ts [hunk 3]
```diff
@@ -86,7 +86,7 @@ describe('Liquidator2', () => {
 
     await tru.initialize()
     await stkTru.initialize(tru.address, pool.address, pool.address, distributor.address, liquidator.address)
-    await lender.initialize(stkTru.address, poolFactory.address, rater.address, AddressZero, pool.address)
+    await lender.initialize(stkTru.address, poolFactory.address, rater.address, AddressZero)
```
Comment (yuchenlintt) [inline test/truefi2/Liquidator2.test.ts:89]: Is this fixing build?

**Pull request merged**